src/index.js

```diff
@@ -3,6 +3,9 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import './css/font-awesome.min.css'
+import './css/materialdesignicons.min.css';
+import './css/style_1.css';
```

Comment:
@leonardnjura, we want to be able to import `.scss` or `.sass` files. I can only see `.css` files used in the project. 🤔

Reply:
@johnmutuma5 thanks for the observation. The first two `.css` files highlighted are for UI icons and we cannot compile these with sass. The last `.css` file highlighted is the compiled version. The directory `~src/css` has the `.sass` or `.scss` creating the file imported on line `8`

Reply:
@johnmutuma5 every `$ npm start` listens to the css directory for changes in the `.sass` or `.scss` and updates/compiles into the repective `.css`

Reply:
@leonardnjura, does that mean that we will be keeping both types, a `.css` and a `.sass` file all containing similar rules but different syntax? 🤔

Reply:
Yes. Perhaps to add light @johnmutuma5  i added a hot reload on css with `npm run build-css` so if we go by the fact that `.sass` or `.scss`allows us to code minimal and it auto compiles into a pretty css

Reply:
These scripts are inside of bin directory after i refactore some clean up.

Reply:
@leonardnjura, I don't think that we really should keep different types of the same `css` styling rules. Is there a way that the current set up can only have `.sass` or `.scss` and not include a `.css` version of the same. Ideally, we should only have one.